webui/master/src/containers/pages/Masters/Masters.test.tsx

```diff
@@ -0,0 +1,59 @@
+/*
+ * The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+ * (the "License"). You may not use this work except in compliance with the License, which is
+ * available at www.apache.org/licenses/LICENSE-2.0
+ *
+ * This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied, as more fully set forth in the License.
+ *
+ * See the NOTICE file distributed with this work for information regarding copyright ownership.
+ */
+
+import { configure, shallow, ShallowWrapper } from 'enzyme';
+import Adapter from 'enzyme-adapter-react-16';
+import { createBrowserHistory, History, LocationState } from 'history';
+import React from 'react';
+import sinon from 'sinon';
+
+import { initialState } from '../../../store';
+import { initialInitState } from '../../../store/init/reducer';
+import { AllProps, MastersPresenter } from './Masters';
+import { routePaths } from '../../../constants';
+import { createAlertErrors } from '@alluxio/common-ui/src/utilities';
+
+configure({ adapter: new Adapter() });
+
+describe('Masters', () => {
+  let history: History<LocationState>;
+  let props: AllProps;
+
+  beforeAll(() => {
+    history = createBrowserHistory({ keyLength: 0 });
+    history.push(routePaths.masters);
+    props = {
+      initData: initialInitState.data,
+      mastersData: initialState.masters.data,
+      errors: createAlertErrors(false),
+      loading: false,
+      refresh: initialState.refresh.data,
+      class: '',
+      fetchRequest: sinon.spy(() => {}),
+    };
+  });
+
+  afterEach(() => {
+    sinon.restore();
+  });
+
+  describe('Shallow component', () => {
```

Comment:
- can you create a test case for when `initData.debug` is `true`?
- can you create test case for some example master data?

Reply:
@lucaspeng12138